hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -1077,6 +1244,12 @@ withRetries :: ShrinkRetries -> Property -> Property
 withRetries n =
   mapConfig $ \config -> config { propertyShrinkRetries = n }
 
+-- | Set the target that a property will skip to before it starts to run.
+--
+withSkip :: Skip -> Property -> Property
+withSkip s =
+  mapConfig $ \config -> config { propertySkip = Just s }
```

Comment:
There's no way to set this back to `Nothing`. I can't immediately think why that would be useful, but do we want it anyway? If we have `withSkipStr` then I think `withSkip` could reasonably take a `Maybe Skip`.

Reply:
nah I think you just wouldn't include the line